java-client/src/main/java/org/opensearch/client/transport/httpclient5/ResponseException.java

```diff
@@ -92,4 +92,12 @@ static String buildMessage(Response response) throws IOException {
     public Response getResponse() {
         return response;
     }
+
+    /**
+     * Status code returned by Elasticsearch. Shortcut for
```

Comment:
```suggestion
     * Status code returned by Opensearch. Shortcut for
```

Reply:
capitalize OpenSearch accordingly.=

Reply:
that was a copy & paste thing (it still says `Elasticsearch` on `OpenSearchException` where i copied this from and i hadn't seen it). i've checked and there are more such places, thus i've now raised #757 to get them all fixed (i'll fix it here as it's new code but won't touch `OpenSearchException` as part of this PR)